cmd/controller/controller.go

```diff
@@ -46,6 +50,8 @@ const (
 	NDMUnknown = "Unknown"
 	// NDMDiskTypeKey specifies the type of disk.
 	NDMDiskTypeKey = "ndm.io/disk-type"
+	// NDMUnmanagedDiskKey specifies disk cr should be managed by ndm or not.
+	NDMUnmanagedKey = "ndm.io/unmanaged"
```

Comment:
"ndm.io/managed?" would improve readability(IMO). Need to see '?' allowed in CR or not.

Reply:
Same I was discussed with @kmova. 
IMO `ndm.io/managed` is good.